.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Hedera Services CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:latest
```

Comment:
Do we need postgres if we're only running the mvn install command and not running Hedera?

Reply:
For now we are not able to skip this part because some tests depends on postgres.